src/transformers/utils/quantization_config.py

```diff
@@ -567,9 +577,7 @@ def post_init(self):
             )
 
         if self.version not in [AWQLinearVersion.GEMM, AWQLinearVersion.GEMV]:
```

Comment:
These are actually doing two different checks. One is checking that its supported in the logic of this class. The other (in the enum) is just checking the enum value. We should do the conversion to the lowercase enum first and then check if it's supported. 

```suggestion
        self.version = AWQLinearVersion.from_str(self.version)
        if self.version not in [AWQLinearVersion.GEMM, AWQLinearVersion.GEMV]:
            raise ValueError(
                f"Only supported versions are in [AWQLinearVersion.GEMM, AWQLinearVersion.GEMV] - not recognized version {self.version}"
            )
```

Reply:
Makes sense!